Signal/src/ViewControllers/ConversationView/ConversationViewItem.h

```diff
@@ -17,6 +17,7 @@ typedef NS_ENUM(NSInteger, OWSMessageCellType) {
     OWSMessageCellType_Video,
     OWSMessageCellType_GenericAttachment,
     OWSMessageCellType_DownloadingAttachment,
+    OWSMessageCellType_ShareContact,
```

Comment:
naming nit (TIOLI): `ShareContact` sounds like a verb, e.g. I'd expect interacting with this cell would cause me to share out a contact.

How about `ContactShare` or just `Contact`

Reply:
Done.